### PR TITLE
feat(start): direct user to admin interface

### LINKED ISSUE #665: Direct the user to /ghost at the end of setup
## This issue is a

* [x] Feature Request
* [ ] Bug Report

### Summary

After completing `ghost install`, I'm like "Now what?". Fortunately I'd seen somewhere else that you can access the admin UI at /ghost, so I appended that to my blog's URL.

But new users might not know this.

### Steps to Reproduce (for a bug report)

1. Complete the [production install steps](https://docs.ghost.org/docs/install)

### Log file (for a bug report)

The setup leaves the user with this:

```
✔ Starting Ghost
You can access your blog at http://myblog.com/

Ghost uses direct mail by default
To set up an alternative email method read our docs at https://docs.ghost.org/docs/mail-config
```

### Technical details (for a bug report)

    Ghost-CLI Version: 1.5.2
    Environment: production
    Command: 'ghost install'

### PR DESCRIPTION
**Pull request by AileenCGN**
closes #665

- added a new message that shows the URL to the admin interface
- uses custom admin URL, if set in config
- shows a slightly different message when rendered after the `install` process

### CODE REVIEW

File: lib/commands/start.js [hunk 1]
```diff
@@ -59,8 +60,20 @@ class StartCommand extends Command {
                     });
                 }).then(() => {
                     if (!argv.quiet) {
+                        const isInstall = argv._ && argv._.length && argv._[0] === 'install';
```
Comment (AileenCGN) [inline lib/commands/start.js]: @acburdine is there a better way to detect the original command that was used here? I couldn't find a better way, but maybe you know one.

**Comment by AileenCGN** [comment]
@peterzimon requesting your feedback wrt the wording.

We'll render this after a new install:
`Next, go to https://somewebsite.com/ghost/ to complete your blog setup`

image

and this when ghost is starting every time:
`Your admin interface is located at https://somewebsite.com/ghost/`

image
_with custom admin URL_

image
_standard, without custom admin URL_

File: lib/commands/start.js [hunk 2]
```diff
@@ -59,8 +60,20 @@ class StartCommand extends Command {
...
+                        let adminUrl = instance.config.get('admin.url') ? instance.config.get('admin.url') : instance.config.get('url');
```
Comment (acburdine) [inline lib/commands/start.js]: you should just be able to do `let adminUrl = instance.config.get('admin.url') || instance.config.get('url');` here

File: lib/commands/start.js (hunk 1 above)
Comment (acburdine) [reply lib/commands/start.js]: hmm - you could check `process.argv[2]` - based on how ghost-cli starts up a value is guaranteed to exist there.

**Comment by peterzimon** [comment]
@AileenCGN 
What about the following text after fresh install:
`Next, go to to your admin interface at https://somewebsite.com/ghost/ to complete your blog setup`

File: lib/commands/start.js (hunk 2 above)
Comment (AileenCGN) [reply lib/commands/start.js]: 👍

**Comment by AileenCGN** [comment]
@peterzimon blog or publication?

File: lib/commands/start.js (hunk 1 above)
Comment (AileenCGN) [reply lib/commands/start.js]: Worked like a charm!

**Comment by AileenCGN** [comment]
@peterzimon This is how it looks like in white, with the URLs highlighted in cyan:

image

**Comment by AileenCGN** [comment]
Text is updated now to:

`Next, go to your admin interface at https://somewebsite.com/ghost/ to complete the setup of your publication` for a fresh install and 

`You can access your publication at https://somewebsite.com/ghost/` for every other start.

**Comment by AileenCGN** [comment]
@acburdine This is ready to go and confirmed with @peterzimon! 🎉

**Pull request merged**